### ISSUE TITLE
[BUG] Hidden fields inside repeaters are broken

### ISSUE THREAD

**Issue by Yceforp** [bolt/bolt#5933]
Saving a record that has an hidden field inside a repeater throws the following exception : 

> DBALException in DBALException.php line 240: 
> Unknown column type "hidden" requested. Any Doctrine type that you use has to be registered with \Doctrine\DBAL\Types\Type::addType(). You can get a list of all the known types with \Doctrine\DBAL\Types\Type::getTypesMap(). If this error occurs during database introspection then you might have forgot to register all database types for a Doctrine Type. Use AbstractPlatform#registerDoctrineTypeMapping() or have your custom types implement Type#getMappedDatabaseTypes(). If the type name is empty you might have a problem with the cache or forgot some mapping information.

Details
-------

 - Relevant Bolt Version: 3.2.0-beta6

Reproduction
------------

Install a new bolt project and use the following contenttypes.yml : 

```
releases:
    name: Releases
    singular_name: Release
    fields:
        title:
            type: text
            class: large
        released_at:
            type: date
            label: Release date
        teaser:
            type: markdown
            height: 150px
        body:
            type: markdown
            height: 300px
        changes:
            type: repeater
            fields: 
                repository: 
                    type: select
                    values: [api, website]
                revision:
                    type: integer
                log:
                    type: hidden
    taxonomy: [ categories ]
    recordsperpage: 100

```

Create a new record, then save : the exception is thrown.

**Comment by GawainLynch** [comment]
@bobdenotter flagging this for 3.2 if @rossriley is able to get to it today

**Comment by bobdenotter** [comment]
fixed by #5938
